docs/contributor_guide/developer_call_notes/2020_07_09.rst

```diff
@@ -46,7 +46,7 @@ Agenda:
 - Outreach: So far the philosophy was to wait for project to stablize before
   major outreach efforts. Most of the participants felt like it's time to do
   more on this (while ensuring we
-  `speak about fairness in the right way <https://fairlearn.github.io/contributor_guide/how_to_talk_about_fairness.html>`_).
+  `speak about fairness in the right way <https://fairlearn.org/contributor_guide/how_to_talk_about_fairness.html>`_).
```

Comment:
Similar to previous comment, I think this should be https://fairlearn.org/main/contributor_guide/how_to_talk_about_fairness.html

Reply:
This file is now deleted thanks to #745